lib/utils.ts

```diff
@@ -1,6 +1,15 @@
 import { type ClassValue, clsx } from "clsx";
 import { twMerge } from "tailwind-merge";
+import { ZoomApiWrapper } from "@/lib/zoomapi";
 
 export function cn(...inputs: ClassValue[]) {
   return twMerge(clsx(inputs));
 }
+
+export async function getZoomApi(): Promise<ZoomApiWrapper> {
+  const zoomImport = process.env.NEXT_PUBLIC_MOCK_ZOOM_API
+    ? import("@/lib/fakezoomapi")
+    : import("@/lib/zoomapi");
+  const zoomModule = await zoomImport;
+  return zoomModule.zoomApi;
+}
```

Comment:
I'm a bit suspicious of files named something too generic, like `util`. Maybe put this in its own file?